tests/unit/test_notebooks_gpu.py

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I think we should add as well smoke and integration tests

Reply:
Working on that.

Reply:
Added